### ISSUE TITLE
Problem with polymorphic models

### ISSUE THREAD

**Issue by rfleschenberg** [jrief/django-admin-sortable2#131]
When used in combination with django-polymorphic, django-admin-sortable2 calculates the order incorrectly when inserting a new instance. This is because it operates on the child model in `SortableAdminMixin.get_max_order`:

```
(Pdb) self.model
<class 'stofferia.models.commodity.Commodity'>
(Pdb) self.model.objects.aggregate(max_order=Max(self.default_order_field))['max_order']
395

(Pdb) Product.objects.aggregate(max_order=Max(self.default_order_field))['max_order']
1680
```

Maybe this can be fixed outside of django-admin-sortable2, for example by using a custom child class.

**Comment by rfleschenberg** [comment]
A simple fix might be something like this:

```python
class PolymorphicSortableAdminMixin(SortableAdminMixin):

    def get_max_order(self, request, obj=None):
        return self.base_model.objects.aggregate(
            max_order=Max(self.default_order_field)
        )['max_order'] or 0



```

**Comment by jrief** [comment]
Instead of using the MAX aggregate, maybe we could try with ``...latest(self.default_order_field)``.

**Comment by rfleschenberg** [comment]
Using `latest()` doesn't seem quite right to me. What if the model we are ordering doesn't have a date field? What about several instances having the same date?

What do you think about shipping the mix-in I sketched above, plus a note that polymorphic users should use it?

**Comment by jrief** [comment]
Yes, I agree to ship that mixin and add it to the docs.
You want to create a PR for it?

**Comment by rfleschenberg** [comment]
Will do. Feel free to remind me if I don't file the PR within a few days, my to-do list is growing and growing ;) This should be fairly easy, so if anyone else picks it up, all the better.

**Comment by jrief** [comment]
Fixed in 0.6.7
Thanks for your attention
